build.conf.arm

```diff
@@ -0,0 +1 @@
+COMPILED_KERNEL_URL=https://github.com/rancher/os-kernel/releases/download/HypriotOS-4.1.12-v7+/linux-4.1.12-hypriotos-v7.-arm.tar.gz
```

Comment:
Yes, just asked.
Then, in the future, I think it's best to make an own kernel that has exactly everything RancherOS needs (and doesn't need), but I agree it's best to start with hypriot